src/Http/GraphQLHandler.php

```
@@ -0,0 +1,211 @@
+     * Decode and validate the incoming JSON payload.
+     *
+     * @return array<string, mixed>
+     *
+     * @throws \InvalidArgumentException when payload is invalid
+     */
+    private function decodeJsonPayload(string $rawBody): array
+    {
+        $payload = json_decode($rawBody, true);
+
+        if (! is_array($payload)) {
+            throw new \InvalidArgumentException('Request body must be a JSON object.');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '96', column '23'). (undefined)

(MissingImport)